### DISCUSSION TITLE
Custom serializer for field in an OutSchema

### DISCUSSION THREAD

**Discussion by twentyforty** [vitalik/django-ninja#246] [Q&A]
I'm using GeoDjango and I have a serializer that I would like to use that produces standards-based GeoJson. I would like to use the django-ninja serialization for all fields except one where I provide my own serializer. I'd like to avoid writing custom serialization code if possible.

**I have models similar to:**

```
class Boundary(models.Model):
    ....
    poly = models.PolygonField().    # gis framework provide field. Basically a list of lat, lng coordinates.

class Place(models.Model):
    ....
    name = models.CharField(max_length=100)
    boundary = OneToOneField(Boundary, on_delete=CASCADE, null=True, blank=True)
```

**I would like to provide a Schema like this:**

```
class PlaceOut(Schema):
    name: str
    boundary: Optional[GeoFeatureOut] = Field(alias="boundary__poly", serializer=GeoFeatureModelSerializer)

class GeoGeometryOut(Schema):
    type: str
    coordinates: List[List[Tuple[float, float]]]

class GeoFeatureOut(Schema):
    type: str
    geometry: GeoGeometryOut

```

Is something like this currently possible? If not, is there any plan to provide custom serialization for specific fields?

Thank you.

**Comment by stephenrauch** [comment]
That functionality is primarily handled by Pydantic.  What you are after might be handled here:

https://pydantic-docs.helpmanual.io/usage/exporting_models/#json_encoders
